app/editor/src/features/storage/StorageListView.tsx

```diff
@@ -71,13 +78,20 @@ export const StorageListView: React.FC = (props) => {
       <li key={i.name}>
         <Row nowrap>
           <Row flex="2 1 0">
-            <FaPhotoVideo />
+            {!i.name.endsWith('.jpg') ? <FaPhotoVideo /> : <FaRegImage />}
```

Comment:
It would be better to make a generic function in our `/utils` folder to determine whether it should have a play button.  then we can add additional file extensions.

Reply:
Updated, thanks